Automattic/Gravatar-SDK-Android
# Update REST API with interest

Closes #182 

### Description

Gravatar has recently added interest to its REST API, so this PR updates the models to retrieve the new field.

<img width="250" alt="image" src="https://github.com/Automattic/Gravatar-SDK-Android/assets/6974554/0ee51321-627e-49c2-8116-51eec251ce99">

### Testing Steps

**Note:** The `interest` field is only provided in authenticated requests.

1. Run the demoApp with a valid API key
2. Retrieve a profile with interests and verify you can see them in the `Raw Profile`
3. Retrieve a profile without interest and verify you can see an empty array
4. Run the demoApp without an API key
5. Retrieve a profile with interests and verify you can't see them. You'll see the field as `null`.
6. Retrieve a profile without interests and verify you can't see them. You'll see the field as `null`.
